prefix_finder/frontend/templates/results.html

```diff
@@ -1,31 +1,63 @@
 {% load static %}
 {% load results %}
 
+<div class="row">
 {% for result in results %}
+  <div class="col-xs-12 col-md-6">
   <div class="panel panel-default">
-    <div class="panel-heading">
-      <h4><a href="{% url 'list' result.code %}" data-toggle="tooltip" data-placement="top" title="More info">{{result.name.en}} ({{result.code}})</a></h4>
+    <div class="panel-heading"><span class="badge pull-right" data-toggle="popover" data-placement="bottom" title="Quality score" data-html="true" data-content="{%for key,value in result.quality_explained.items %}{{ key }} - {{ value }}<br/>{%endfor%}" data-trigger="hover">Quality score: {{ result.quality }}</span>
```

Comment:
Some missing spaces between `{%` and the following keyword (eg. `{%for key ...`)